src/core/package.ts

```diff
@@ -0,0 +1,46 @@
+import Table from 'cli-table3';
+import chalk from 'chalk';
+import FieldLoader from '../util/fieldLoader';
+import Differ from '../util/differ';
+
+class Package {
+  fieldLoader = new FieldLoader();
+  differ = new Differ();
+
+  async runLocalCheck(): Promise<void> {
```

Comment:
I think we should keep with the `run` method name to keep things similar between commands:

```
Licenses.run
Package.run
FooBar.run
```
For different flows we can use parameters.

Reply:
At first, I thought the same thing but later I decided to do it this way for better clarity. (I can totally change that though)